### PR TITLE
Added functionality to Zoom class

### PR DESCRIPTION
**Pull request by bzupnick**
This is a PR for [this issue](https://github.com/move-coop/parsons/issues/459).

This is my first PR here so let me know if I missed something!

### CODE REVIEW

**Comment by eliotst** [comment]
Thanks so much for this, @bzupnick ! We will take a look, and we should be able to provide a review in the next couple of days.

File: parsons/zoom/zoom.py [hunk 1]
```diff
@@ -153,3 +153,67 @@ def get_past_meeting_participants(self, meeting_id):
         tbl = self._get_request(f'report/meetings/{meeting_id}/participants', 'participants')
         logger.info(f'Retrieved {tbl.num_rows} participants.')
         return tbl
+
+    def get_meeting_registrants(self, meeting_id):
+        """
+        Get meeting registrants
```
Comment (jburchard) [inline parsons/zoom/zoom.py:159]: Add a period.*

* We could probably stand to have a style guide for documentation at some point.

File: parsons/zoom/zoom.py [hunk 2]
```diff
@@ -153,3 +153,67 @@ def get_past_meeting_participants(self, meeting_id):
...
+                See :ref:`parsons-table` for output options.
+        """
+
+        tbl = self._get_request(f'meetings/{meeting_id}/registrants', 'registrants')
+        logger.info(f'Retrieved {tbl.num_rows} registrants.')
+        return tbl
+
+    def get_user_webinars(self, user_id):
+        """
+        Get meeting registrants
```
Comment (jburchard) [inline parsons/zoom/zoom.py:175]: Period.

File: parsons/zoom/zoom.py [hunk 3]
```diff
@@ -153,3 +153,67 @@ def get_past_meeting_participants(self, meeting_id):
...
+                See :ref:`parsons-table` for output options.
+        """
+
+        tbl = self._get_request(f'users/{user_id}/webinars', 'webinars')
+        logger.info(f'Retrieved {tbl.num_rows} webinars.')
+        return tbl
+
+    def get_past_webinar_participants(self, webinar_id):
+        """
+        Get past meeting participants
```
Comment (jburchard) [inline parsons/zoom/zoom.py:191]: Period.

**Review by jburchard — CHANGES_REQUESTED** [review]
This is really great! Thank you.

Just a little minor doc styling to change and then we are good to go!

**Comment by bzupnick** [comment]
migrated over to https://github.com/move-coop/parsons/pull/461

**Pull request closed without merging**